src/pal/tools/cppmunge/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ project(cppmunge)
 
 add_definitions(-D_BSD_SOURCE) # Needed to get realpath part of CRT for c99 compat.
 
+if(CLR_CMAKE_PLATFORM_LINUX)
+  add_definitions(-D__LINUX__=1)
```

Comment:
If it's dead code anyway, I honestly don't think it's worth redoing this part of the PR. And until the PR to delete it is merged, this is still something stopping the build for us working on FreeBSD, so I'd like to keep it in.